src/main/java/frc/robot/subsystems/vision/CameraPoses.java

```diff
@@ -5,33 +5,25 @@
 import edu.wpi.first.math.geometry.Translation3d;
 import edu.wpi.first.math.util.Units;
 
-/** Configuration class containing the poses of all cameras on the robot for vision processing. */
 public class CameraPoses {
-
-  private CameraPoses() {
-    throw new IllegalStateException("Utility class");
-  }
-
   public static final Pose3d[] cameraPoses =
       new Pose3d[] {
         // Front Left
         new Pose3d(
-            new Translation3d(0.235, 0.235, 0.267), // Right camera translation (X, Y, Z)
-            new Rotation3d(0.0, Units.degreesToRadians(-12.63), Units.degreesToRadians(45))),
+            new Translation3d(0.256, 0.213, 0.267), // Right camera translation (X, Y, Z)
+            new Rotation3d(0.0, Units.degreesToRadians(-12.63), Units.degreesToRadians(-15))),
         // Front Right
         new Pose3d(
-            new Translation3d(0.235, -0.235, 0.267), // Left camera translation (X, Y, Z)
+            new Translation3d(0.268, -0.213, 0.267), // Left camera translation (X, Y, Z)
             new Rotation3d(
-                0.0,
-                Units.degreesToRadians(-12.63),
-                Units.degreesToRadians(-45))), // in radians btw
+                0.0, Units.degreesToRadians(-12.63), Units.degreesToRadians(45))), // in radians btw
         //        // Back Right
         new Pose3d(
             new Translation3d(-0.235, -0.235, 0.267), // Left camera translation (X, Y, Z)
             new Rotation3d(
                 0.0,
                 Units.degreesToRadians(-12.63),
-                Units.degreesToRadians(-45.0 - 90))), // in radians btw
+                Units.degreesToRadians(-45 - 90))), // in radians btw
```

Comment:
Per Sonar you want one of these as a double so that floating point processing occurs.